test/WidenImage.js

```diff
@@ -0,0 +1,78 @@
+import applib from '../build/applib'
+import assert from 'assert'
+import fixtureIO from './Utilities/FixtureIO'
+import styleMocking from './Utilities/StyleMocking'
+
+const maybeWidenImage = applib.WidenImage.maybeWidenImage
+let document = null
```

Comment:
I think I'm pretty against top level vars if they can be avoided. Perhaps less concerned in test files like this, but your `this.document` in `beforeEach()` suggestion should achieve same effect - I'll try switching to that.